examples/service_accounts.tf

```diff
@@ -1,3 +1,9 @@
+provider "time" {}
+resource "time_rotating" "ninety_days" {
+    rotation_days = 90
+}
+
 resource "prefect_service_account" "sa_example" {
-    name = "beautiful-service-account-example"
+    name = "my-service-account"
+    api_key_expiration = time_rotating.ninety_days.rotation_rfc3339
```

Comment:
this is the way we'll recommend practitioners handle key rotations, since the `time_rotating` resource handles properly exporting an rfc3339 compliant datetime string